source/fundamentals/enterprise-authentication.txt

```diff
@@ -2,4 +2,261 @@
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using the
+**authentication mechanisms** available only in the {+mongo-enterprise+}.
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
+
+You can use the following authentication mechanisms with the latest version of
+{+mongo-enterprise+}.
+
+- :ref:`csharp-kerberos`
+- :ref:`csharp-LDAP`
+
+To authenticate using another mechanism, see the
+:ref:`Authentication Mechanisms guide <csharp-authentication-mechanisms>`. For
+more information on establishing a connection to your MongoDB cluster, see the
+:ref:`csharp-connect-to-mongodb`.
+
+Specify an Authentication Mechanism
+-----------------------------------
+
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using either of the following methods:
+
+- A **connection string**, also known as a **connection URI**, which is a string
+  that tells the driver how to connect to a MongoDB deployment and how to behave while
+  connected.
+
+- A factory method for the supported authentication mechanism, contained in the 
+  ``MongoCredential`` class.
+.. _csharp-kerberos:
+
+GSSAPI/Kerberos
+---------------
+
+The Generic Security Services API (``GSSAPI``) authentication mechanism allows the 
+user to authenticate to a Kerberos service using the user's principal name.
+
+The following examples specify the authentication mechanism using the following
+placeholders:
+
+- ``<username>``: Your URL-encoded principal name, e.g. "username%40REALM.ME"
+- ``<password>``: Your Kerberos user's password
+- ``<hostname>``: The network address of your MongoDB server, accessible by your client
+
+Select the :guilabel:`Connection String` or :guilabel:`MongoCredential` tab to
+see the corresponding syntax for specifying the ``GSSAPI/Kerberos`` authentication
+mechanism:
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: connection-string
+
+      .. code-block:: csharp
+
+         var mongoClient = new MongoClient("mongodb://<username>:<password>@<hostname>/?authMechanism=GSSAPI");
+
+   .. tab:: MongoCredential
+      :tabid: mongo-credential
+
+      .. code-block:: csharp
+         :emphasize-lines: 1
+
+         var credential = MongoCredential.CreateGssapiCredential("<username>", "<password>");
+         var settings = MongoClientSettings.FromConnectionString("<connection string>");
+         settings.Credential = credentials;
+         var mongoClient = new MongoClient(settings);
+
+.. tip:: Omit Password
+
+   You can omit the password if one of the following are true:
+
+   - On Windows, the process owner running the application is the same as the
+     user needing authentication.
+
+   - ON Linux, the user has initialized their keytab via ``kinit username@REALM.COM``.
```

Comment:
Is "REALM.COM" all caps for a reason?

```suggestion
   - On Linux, the user has initialized their keytab via ``kinit username@realm.com``.
```

Reply:
It looks like that's [convention](https://docs.oracle.com/cd/E21455_01/common/tutorials/kerberos_principal.html) for Kerberos realm names.